app/controllers/idv/document_capture_controller.rb

```diff
@@ -49,7 +49,6 @@ def extra_view_variables
         flow_path: 'standard',
         sp_name: decorated_sp_session.sp_name,
         failure_to_proof_url: return_to_sp_failure_to_proof_url(step: 'document_capture'),
-        skip_doc_auth: idv_session.skip_doc_auth,
```

Comment:
@KeithNava 
(I’m starting this general conversation on a specific file so that replies are threaded.)

When I look at the “Progress” section of the ticket LG-13006, I see checkbox 4 
- [ ] remove decision points with the old value skip_doc_auth and replace with skip_doc_auth_from_how_to_verify. PR created (23 May), wait for next deploy to merge.

After the “wait for deploy” line, I see checkbox 5

- [ ] remove skip_doc_auth from everywhere, except session.rb.

Based on the "Summary of changes" in the PR description, I _think_ this PR is meant to take care of checkbox 5, correct?  I don’t see that Micah N’s PR #10690, which I believe took care of checkbox 4, was ever merged.  Are you planning on merging that PR before this one?  Does it need a review?

Let me know, thanks!  I haven’t yet had a chance to review this PR, but I wanted to ask this question upfront because I know sequencing gets tricky for PRs that are affected by 50-50 state.

Reply:
@KeithNava I have to plead some level of ignorance here. 😇 

I wasn't _intentionally_ changing anything relating to `skip_doc_auth` in that PR; I just removed it as appearing to be unused. [This comment](https://github.com/18F/identity-idp/pull/10682/files#r1610564168) paints some context for the change&mdash;we were passing that value into the constructor but didn't seem to ever be using its value in the `show` method, so I removed it as dead code. (TBH I don't completely remember why I thought it was unused, but it seems like people agreed with me. I've lost some of this context to time.)

I didn't have Micah's PR in mind when I worked on this. It looks like his change to this file would be unnecessary, but he did also update the HybridHandoffController which I didn't touch, so that likely still needs attention.